Updates for PM review. (#35)
mulesoft/docs-partner-manager: modules/ROOT/pages/setup.adoc

endif::[]

Set up Anypoint Partner Manager to be used by different people in your organization: message flow developers, operations managers, and IT managers.
To configure the system, you should be familiar with these concepts before setting up Partner Manager:

* EDI message formats and schemas

* Dataweave transformations

* Digital Certificates such as X.509 certificates in PKCS12 file format for the AS2 transport protocol

* Configuration of protcols such as AS2, SFTP, and HTTPS

* The concept of organizations and environments in the Anypoint platform in context to Access Management and Mule Runtime Manager

== Minimum Requirements

Verify that you have the following software and other requirements:

* Mule runtime engine 4.1.1 or higher. Make sure you have the needed Mule runtime Core/vCore capacity to handle your B2B transaction workload needs

If you are unsure about the capacity you need to handle your B2B transactions, please contact your MuleSoft account representative for assistance.

* A storage solution to store the B2B transmission payloads your send or receive. Storage options can include a Database, Amazon S3, Azure Blob Storage, and so on.

> * Anypoint Studio 7.1 or higher to create DataWeave maps

Reviewer: ```suggestion
* Anypoint Studio 7.1 or later to create DataWeave maps
```

Already edited this above. I am confused by repeated text. Please do not include it in PRs.

> Set up Partner Manager to be used by different people in your organization: message flow developers, operations managers, and IT managers.

Reviewer: Where does this file fit in the doc chronology? Is this introduced in the introduction? I don't recall seeing this task in what is covered by this doc.

Reviewer: This belongs at the beginning of this doc. Why is it here? Did you just add it?

> To configure the system, you should be familiar with these concepts before setting up Partner Manager:

Reviewer: ```suggestion
You should be familiar with these concepts before setting up Partner Manager:
```

* EDI message formats and schemas

* Dataweave transformations

> * Digital Certificates such as X.509 certificates in PKCS12 file format for the AS2 transport protocol

Reviewer: ```suggestion
* Digital certificates, such as X.509 certificates in PKCS12 file format for the AS2 transport protocol
```

* Configuration of protcols such as AS2, SFTP, HTTPS

> * Understanding of the concept of organizations and environments in the Anypoint platform in context to Access Management and Mule Runtime Manager

Reviewer: ```suggestion
* Organizations and environments in Anypoint Platform in relation to access management and Anypoint Runtime Manager
```

== Minimum Requirements

Verify that you have the following software and other requirements:

> * Mule runtime engine 4.1.1 or higher. Make sure you have the needed Mule runtime Core/vCore capacity to handle your B2B transaction workload needs

Reviewer: ```suggestion
* Mule runtime engine 4.1.1 or later. 
  Ensure that you have the Mule runtime Core/vCore capacity required to process your B2B transaction workload. If you are uncertain, contact your MuleSoft representative.
```

> If you are unsure about the capacity you need to handle your B2B transactions, please contact your MuleSoft account representative for assistance.

Reviewer: omit

> * A storage solution to store the B2B transmission payloads your send or receive. Storage options can include a Database, Amazon S3, Azure Blob Storage, and so on.

Reviewer: ```suggestion
* A storage solution capable of storing your anticipated B2B transmission payloads. 
Storage options can include a database, Amazon S3, Azure Blob Storage, and so on.
```

== Environment Configuration

> You'll need to set up and configure your environment before you begin using Partner Manager.

Reviewer: Sounds like it should be just a list item in prerequisites, with a link in See Also.

> Please refer to Anypoint platform documentation to learn more about https://docs.mulesoft.com/access-management/environments[configuring your Anypoint platform environments].

Reviewer: ditto

* Partner Manager Admin (Sandbox) - Can perform all operations across Partner Manager for all Sandbox environments.

. Verify that you have access to Partner Manager.

> This means that you must have the required entitlements and have assigned the `Partner Manager Admin` role in Access Management for the environment.

Reviewer: What is an "entitlement"? Could this simply be You must have the Partner Manager Admin role for access management?

GLOBAL: Do not capitalize access management.

Official name | access management | As a capability, this is a descriptor of the functionality. These are predetermined names for capabilities, so while they are not a name, they should be described consistently
-- | -- | --

> . Setup the required Storage API. The storage API should be accessible from the Mule Runtime network.

Reviewer: ```suggestion
. Set up the required storage API. 
  The storage API should be accessible from the Mule network.
```

> === Setup the Payload Storage API

Reviewer: GLOBAL: Search for "setup" and evaluate for correct presentation.

setup = noun, adjective
set up = verb

Anypoint Partner Manager enables you to connect your own storage solution (such as a database, Amazon S3, Azure Blob Storage, and so on) to store the transmission content such as incoming and outgoing B2B message payloads. You will need to establish an API that provides Anypoint Partner Manager with a standardized way to connect to your unique storage. To setup the Storage API, follow the steps indicated below.

> 

Reviewer: This looks like text that already exists above.
NOT EDITED
